src/modules/dashboard/components/PermissionManagementDialog/PermissionManagementCheckbox.tsx

```diff
@@ -1,59 +1,72 @@
 import React, { useMemo } from 'react';
 import { defineMessages, FormattedMessage, useIntl } from 'react-intl';
-import { ColonyRole } from '@colony/colony-js';
+import { ColonyRole, ROOT_DOMAIN_ID } from '@colony/colony-js';
 
 import { Checkbox } from '~core/Fields';
-import Heading from '~core/Heading';
 import Popover from '~core/Popover';
+import PermissionsLabel from '~core/PermissionsLabel';
 
-import styles from './PermissionCheckbox.css';
+import styles from './PermissionManagementCheckbox.css';
+
+const idBase =
+  'dashboard.PermissionManagementDialog.PermissionManagementCheckbox';
 
 const MSG = defineMessages({
   roleWithAsterisk: {
-    id: 'admin.ColonyPermissionEditDialog.roleWithAsterisk',
+    id: `${idBase}.roleWithAsterisk`,
```

Comment:
Please don't do this.

1. I know this is a bit of code repetition, but `react-intl` doesn't handle dynamic ids very well
2. If you really wanted to do this properly, you could have used the `displayName` variable

But again, please type out the whole message descriptor id every time